PythonAPI/test/API/spawn_vehicles.py

```diff
@@ -0,0 +1,58 @@
+
+from __future__ import print_function
+import unittest
+import glob
+import os
+import sys
+try:
+    sys.path.append(glob.glob('../carla/dist/carla-*%d.%d-%s.egg' % (
+        sys.version_info.major,
+        sys.version_info.minor,
+        'win-amd64' if os.name == 'nt' else 'linux-x86_64'))[0])
+except IndexError:
+    pass
+try:
+    sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))) + '/carla')
+except IndexError:
+    pass
+
+import carla
+
+class TestVehiclesSpawnTest(unittest.TestCase):
+    print("IntegrationTest.Vehicles.VehiclesSpawnTest")
```

Comment:
```suggestion
    print("IntegrationTest.Vehicles.VehiclesSpawnTest")
```
This kind of prints are not needed, so rest running engines like pytest add this kind of logs.